README.md

```diff
@@ -14,13 +14,23 @@ master ( buid )
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- feature
 
 ### :camera_flash: Screenshots
+
```

Comment:
First raise this change to develop and then it will reflect in `main` once you sync everything.

Reply:
Done 👍🏻